json-sql

```diff
@@ -84,6 +99,24 @@ class DB:
     def _table_names(self):
         return NotImplementedError
 
+class ORACLEADB(DB):
+    schema = None
+    def __init__(self, user, password, schema, tnsname, lib_dir=None):
+        if os.environ['TNS_ADMIN'] == "":
```

Comment:
I'm not familiar with OracleDB, why does TNS_ADMIN need to be set? Although we check for it, I can't find where TNS_ADMIN is used.

Reply:
`TNS_ADMIN` is required by the cx_Oracle library. It's the path to a "wallet" directory containing keys that are used to secure the connection to the Oracle DB. The library would throw an error if it's not set, but it's a little vague so I wanted to be more explicit.

Reply:
Great thanks, it's not blocking but if you'd like, you can add a comment here mentioning as such so the next reader isn't confused.